ethkey/main.cpp

```diff
@@ -82,25 +67,42 @@ int main(int argc, char** argv)
 	setDefaultOrCLocale();
 	KeyCLI m(KeyCLI::OperationMode::ListBare);
 	g_logVerbosity = 0;
-
-	for (int i = 1; i < argc; ++i)
+	po::options_description generalOptions("General Options");
+	generalOptions.add_options()
+			("verbosity,v", po::value<int>(), "<0 - 9>  Set the log verbosity from 0 to 9 (default: 8).")
+			("version,V", "Show the version and exit.")
+			("help,h",  "Show this help message and exit.");
+	po::parsed_options parsed = po::command_line_parser(argc, argv).options(generalOptions).allow_unregistered().run();
```

Comment:
We should catch exceptions thrown from this, because parsing options can end with errors... See like I did it in https://github.com/ethereum/cpp-ethereum/pull/4639